tests/components/sensor/test_pilight.py

```diff
@@ -0,0 +1,117 @@
+"""The tests for the Pilight sensor platform."""
+import unittest
+from unittest.mock import patch
+
+from homeassistant.bootstrap import _setup_component
+import homeassistant.components.sensor as sensor
+from homeassistant.components import pilight
+
+from tests.common import get_test_home_assistant
+
+
+def fire_pilight_message(hass, protocol, data):
+    """Fire the fake pilight message."""
+    message = {pilight.ATTR_PROTOCOL: protocol}
+    message.update(data)
+    hass.bus.fire(pilight.EVENT, message)
+
+
+class TestSensorPilight(unittest.TestCase):
+    """Test the Pilight sensor."""
+
+    def setUp(self):  # pylint: disable=invalid-name
```

Comment:
The need is that otherwise you cannot simply run this unittest stand alone by typing ` pytest tests/components/sensor/test_pilight.py`